Fix grammar in store-credentials-key-vault.md (#260)
MicrosoftDocs/azure-databases-docs: articles/cosmos-db/store-credentials-key-vault.md

[!INCLUDE[NoSQL, MongoDB, Cassandra, Gremlin, Table](includes/appliesto-nosql-mongodb-cassandra-gremlin-table.md)]

> [!IMPORTANT]

> > It's recommended to access Azure Cosmos DB using a system-assigned managed identity. If both the managed identity solution and cert based solution do not meet your needs, please use the Azure Key vault solution in this article.

Reviewer: ```suggestion
> We recommend that you access Azure Cosmos DB by using a system-assigned managed identity. If both the managed identity solution and cert based solution do not meet your needs, please use the Azure Key vault solution in this article.
```
